scripts/kind-build-test.sh

```diff
@@ -238,21 +248,7 @@ if [[ "$TEST_HELM_CHARTS" == true ]]; then
   $TEST_RELEASE_DIR/test-helm.sh "$AWS_SERVICE" "$VERSION"
 fi
 
-# Fork E2E tests based on bash or Python
-service_test_dir="$TEST_E2E_DIR/$AWS_SERVICE"
-
-if [ ! -d "$service_test_dir" ]; then
-    echo "No tests for service $SERVICE"
-    exit 0
-fi
-
-# check to see if the tests use pytest
-[[ -f "$service_test_dir/__init__.py" ]] && enable_python_tests="true" || enable_python_tests="false"
-
-if [[ "$enable_python_tests" == "false" ]]; then
-    # Run tests directly
-    $TEST_E2E_DIR/run-tests.sh $AWS_SERVICE
-else
-    # Run tests inside Dockerfile
-    $TEST_E2E_DIR/build-run-test-dockerfile.sh $AWS_SERVICE
-fi
+# run e2e tests
+export SKIP_PYTHON_TESTS="$SKIP_PYTHON_TESTS"
+export RUN_PYTEST_LOCALLY="$RUN_PYTEST_LOCALLY"
```

Comment:
nit: These can be simplified down to just `export SKIP_PYTHON_TESTS` and `export RUN_PYTEST_LOCALLY`?

Reply:
Done, thanks for the feedback!